packages/story-editor/src/components/fontCheck/fontCheckDialog.js

```diff
@@ -0,0 +1,194 @@
+/*
+ * Copyright 2022 Google LLC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+/**
+ * External dependencies
+ */
+import PropTypes from 'prop-types';
+import { TranslateWithMarkup, __ } from '@googleforcreators/i18n';
+import { trackClick } from '@googleforcreators/tracking';
+import styled from 'styled-components';
+import { useState, useCallback } from '@googleforcreators/react';
+import {
+  Button,
+  BUTTON_SIZES,
+  BUTTON_TYPES,
+  Text,
+  THEME_CONSTANTS,
+  Icons,
+  Link,
+} from '@googleforcreators/design-system';
+
+/**
+ * Internal dependencies
+ */
+import { useStory } from '../../app';
+import Dialog from '../dialog';
+import FontPicker from '../fontPicker';
+import { useConfig } from '../../app/config';
+import { TEXT_ELEMENT_DEFAULT_FONT } from '../../app/font/defaultFonts';
+
+const DialogContent = styled.div`
+  display: grid;
+  row-gap: 21px;
+`;
+
+const ChevronRight = styled(Icons.ChevronRightSmall)`
+  width: 32px;
+  height: 32px;
+`;
+
+const ChevronLeft = styled(ChevronRight)`
+  transform: rotate(180deg);
+`;
+
+const DashboardLink = styled.span`
+  display: flex;
+  color: black;
+  align-self: flex-start;
+  align-items: center;
+  font-weight: normal;
+  margin-right: 40px;
+`;
+
+const defaultFont = {
+  font: {
+    id: TEXT_ELEMENT_DEFAULT_FONT?.family,
+    ...TEXT_ELEMENT_DEFAULT_FONT,
+  },
+};
+
+export const FontCheckDialog = ({ isOpen, onClose, missingFont }) => {
+  const { dashboardLink, dashboardSettingsLink, isRTL } = useConfig();
+  const [suggestedFont, setSuggestedFont] = useState(defaultFont);
+  const { updateElementsByFontFamily } = useStory((state) => ({
+    updateElementsByFontFamily: state.actions.updateElementsByFontFamily,
+  }));
+
+  /**
+   * Handles updating suggested font which is passed to the FontPicker
+   *
+   * @param {Object} [font] Font object.
+   * @return void
+   */
+  const onChange = useCallback((font) => {
+    setSuggestedFont({ font });
+  }, []);
```

Comment:
Does this really need to wrapped in a `useCallback`?

Reply:
Removed the useCallback